amundsen_application/static/js/components/ColumnList/index.tsx

```diff
@@ -216,6 +229,7 @@ const ColumnList: React.FC<ColumnListProps> = ({
       sort_order: item.sort_order,
       usage: getUsageStat(item),
       stats: hasItemStats ? item.stats[0] : null,
+      badges: hasColumnBadges ? item.badges : [],
```

Comment:
Yeah, I was thinking yesterday how we should do a local check for hasItemBadges if we wanted to default it to null